js/src/modals/CreateAccount/RecoveryPhrase/recoveryPhrase.js

```diff
@@ -136,7 +136,7 @@ export default class RecoveryPhrase extends Component {
       .filter((part) => part.length);
     let recoveryPhraseError = null;
 
-    if (!recoveryPhrase || recoveryPhrase.length < 25 || phraseParts.length < 8) {
+    if (!recoveryPhrase || !recoveryPhrase.length) {
```

Comment:
[tiny grumble] I'd prefer the more explicit `recoveryPhrase.length === 0` here. Same below.